jerry-core/vm/vm.c

```diff
@@ -847,6 +847,31 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
 
         if (opcode_data & VM_OC_BACKWARD_BRANCH)
         {
+#ifdef JERRY_VM_EXEC_STOP
+          if (JERRY_CONTEXT (vm_exec_stop_cb) != NULL)
+          {
+            if (--JERRY_CONTEXT (vm_exec_stop_counter) == 0)
```

Comment:
I'm really tempted to start a potentially long-running debate on "It is mostly undefined". (It is certainly not.) But I wont. Keep this as is.